pkg/executor/executor.go

```diff
@@ -107,7 +107,7 @@ func MakeExecutor(ctx context.Context, logger *zap.Logger, cms *cms.ConfigSecret
 // get specialized. In other words, it ensures that when there's an
 // ongoing request for a certain function, all other requests wait for
 // that request to complete.
-func (executor *Executor) serveCreateFuncServices() {
+func (executor *Executor) serveCreateFuncServices(ctx context.Context) {
```

Comment:
We should not pass context to this function. Every request on the channel should pass its own context and we pass same context further.

```go
createFuncServiceRequest struct {
    context  context.Context
    function *fv1.Function
    respChan chan *createFuncServiceResponse
}
```